manubot/pandoc/cite_filter.py

```diff
@@ -48,6 +53,10 @@
 - `manubot-clear-requests-cache` (boolean):
   If true, clear the requests cache at `manubot-requests-cache-path`.
 
+- `manubot-fail-on-errors` (boolean):
```

Comment:
Is this adding documentation of an existing feature?

Reply:
yes I realized we didn't document this option